tests/data/function_trailing_comma.py

```diff
@@ -21,6 +21,15 @@ def xxxxxxxxxxxxxxxxxxxxxxxxxxxx() -> Set[
 ]:
     json = {"k": {"k2": {"k3": [1,]}}}
 
+def no_trailing_comma_one_argument_short(arg): pass
+def no_trailing_comma_multiple_arguments_short(a, b, c): pass
+def no_trailing_comma_one_argument_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx(
```

Comment:
Needs more test cases:
- One like this but currently on one line
- One like this but with a trailing comma already present

Reply:
Done. Just for reference here's the link to the failed primer run (for `pyanalyze`):
https://github.com/cdce8p/black/runs/3039749494?check_suite_focus=true#step:5:40

As best as I can tell, the change only causes a reformat if black is run with `skip-magic-trailing-comma = true`.

Reply:
Thanks! I don't like this change from the pyanalyze run:
```
         def forward_ref_bad_attribute(
-            x: "collections.defalutdict",  # E: undefined_name
+            x: "collections.defalutdict"  # E: undefined_name
         ):
```
Now if I want to add an argument, I'll also have to insert a comma in the middle instead of just adding a new line

Reply:
That does make sense. Surprisingly easy to add after reverting some of the previous changes. See the last commit.

This here is the latest primer output for `pyanalyze`:
https://github.com/cdce8p/black/runs/3041092246?check_suite_focus=true#step:5:69